daml-assistant/integration-tests/src/Main.hs

```diff
@@ -389,6 +389,48 @@ packagingTests = testGroup "packaging"
             , "dependencies: [daml-prim, daml-stdlib]"
             ]
         withCurrentDirectory projDir $ callCommandQuiet "daml build"
+    , testCase "Dalf imports" $ withTempDir $ \projDir -> do
```

Comment:
Definitely not necessary to do this in this PR but it would be nice to move all (but one) of the `damlc build` tests into a separate packaging test suite that calls `damlc build` instead of `daml build`. The integration tests are a bit annoying since they always have to build the whole SDK tarball and you can’t properly filter tests using `-p` since some of them depend on each other.

Originally that wasn’t possible since `damlc build` used to fail `ghc-pkg` but we’ve fixed that a while back.

Reply:
Agreed.